diplom.py

```diff
@@ -21,14 +21,17 @@ def mean_average_precision(predicted_probability_distribution,
     k = -1
     for row in predicted_probability_distribution:
```

Comment:
I don't understand how to use it
list(enumeratirs(predicted_probability_distribution))
and then each time _next_() ?